open_seq2seq/parts/convs2s/embedding_layer.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 MLBenchmark Group. All Rights Reserved.
```

Comment:
This file is not needed. Use ``EmbeddingSharedWeights`` class from parts/transformer/embedding_layer.py